extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -299,27 +300,18 @@ export class AmpAdXOriginIframeHandler {
    * @param {string} origin
    * @private
    */
-  updateSize_(height, width, source, origin) {
-    // Calculate new width and height of the container to include the padding.
-    // If padding is negative, just use the requested width and height directly.
-    let newHeight, newWidth;
-    height = parseInt(height, 10);
-    if (!isNaN(height)) {
-      newHeight = Math.max(this.element_./*OK*/offsetHeight +
-          height - this.iframe./*OK*/offsetHeight, height);
-    }
-    width = parseInt(width, 10);
-    if (!isNaN(width)) {
-      newWidth = Math.max(this.element_./*OK*/offsetWidth +
-          width - this.iframe./*OK*/offsetWidth, width);
-    }
-    if (newHeight !== undefined || newWidth !== undefined) {
-      this.baseInstance_.attemptChangeSize(newHeight, newWidth).then(() => {
-        this.sendEmbedSizeResponse_(
-          true /* success */, newWidth, newHeight, source, origin);
-      }, () => this.sendEmbedSizeResponse_(
-          false /* success */, newWidth, newHeight, source, origin));
-    }
+  handleResize_(height, width, source, origin) {
+    this.uiHandler_.updateSize(height, width, this.iframe./*OK*/offsetHeight,
```

Comment:
`/*OK*/offsetHeight` is this inside a vsync measure? I realize this is essentially the same code as before, but I've just noticed.